invenio_search/ext.py

```diff
@@ -88,6 +85,8 @@ def _collect_templates(self, entrypoint_group):
             else:
                 result.append(self.register_templates(ep.module_name))
 
+        entry_points(entrypoint_group, func)
```

Comment:
minor: that's the usage pattern I'm somewhat unhappy with since it reads a bit less Pythonic now. If `entry_points` was `map_entry_points` it would make sense, but otherwise I much prefer the procedural `for ep in entry_points(...):` loop.